src/test/java/marquez/api/mappers/NamespaceResponseMapperTest.java

```diff
@@ -44,7 +44,8 @@ public void testSingleNamespace() {
     assertThat(Timestamp.valueOf(mappedResponse.getCreatedAt()))
         .isEqualTo(descriptionNamespace.getCreatedAt());
```

Comment:
Can't directly comment on this b/c it's not part of the review, but we can make all the member variables `private`, `final`, and `static`.